### PR TITLE
fix(config): Enhance ES 2.4 config; make scripts safer

### PR DESCRIPTION
**Pull request by vhf**
Here's the image config that works with the two ES2.4 PRs.

Ready for review.

### CODE REVIEW

File: elasticsearch.yml [hunk 1]
```diff
@@ -15,4 +15,9 @@ path:
 node:
   name: ${HOSTNAME}
 
-script.inline: on
+# https://www.elastic.co/guide/en/elasticsearch/reference/2.4/modules-scripting.html#enable-dynamic-scripting
+script.inline: true
+# https://www.elastic.co/guide/en/elasticsearch/reference/2.4/docs-index_.html#index-creation
+action.auto_create_index: false
+# https://www.elastic.co/guide/en/elasticsearch/reference/2.4/url-access-control.html#url-access-control
+rest.action.multi.allow_explicit_index: false
```
Comment (marcbachmann) [inline elasticsearch.yml]: Thanks

**Comment by marcbachmann** [comment]
@mheiniger please make sure that this configuration is compatible with your setup.
Especially because of the multi-index-query config.

File: elasticsearch.yml (hunk 1 above)
Comment (marcbachmann) [inline elasticsearch.yml]: I wonder whether I told you about that or whether you have the same opinion as  I do?

Comment (vhf) [inline elasticsearch.yml]: You asked me to add this. ;)

Comment (mheiniger) [inline elasticsearch.yml]: What do you think about this: https://www.elastic.co/guide/en/elasticsearch/reference/2.4/dynamic-mapping.html#_disabling_automatic_type_creation ?

**Comment by mheiniger** [comment]
`rest.action.multi.allow_explicit_index: false` breaks 69 tests in the nzz-downstream

**Comment by marcbachmann** [comment]
I didn't hear anything from you anymore. Are you already fixing those? 😆 

@vhf we should either remove the `allow_explicit_index ` config or make it configurable.

@mheiniger there are two ideas behind disallowing multi index queries. 
- By enforcing that we can keep the data structure simple and prevent any relations that make indexing/management more complicated. 
- And we can protect the whole endpoint with a url based authorization proxy that checks the permission of a client. So multiple apps (projects or channels in our case) could run on the same server without any access to other resources.

File: elasticsearch.yml (hunk 1 above)
Comment (vhf) [inline elasticsearch.yml]: No idea. @marcbachmann ?

**Comment by vhf** [comment]
> @vhf we should either remove the allow_explicit_index config or make it configurable.

Done.

**Comment by vhf** [comment]
Still waiting @marcbachmann

**Pull request merged**